lib/launcher.rb

```diff
@@ -40,6 +40,9 @@ def https_app
           ::Proxy::Plugins.enabled_plugins.each {|p| instance_eval(p.https_rackup)}
         end
 
+        ssl_options = OpenSSL::SSL::SSLContext::DEFAULT_PARAMS[:options]
```

Comment:
Should we add `OpenSSL::SSL::OP_NO_COMPRESSION` option too? Looks like it's turned on by default...